src/lime/extension/Main.hx

```diff
@@ -191,15 +192,22 @@ class Main {
 		var task = new Task (definition, name, "lime");
 		
 		task.execution = new ShellExecution ("lime " + args.join (" "), { cwd: workspace.rootPath, env: haxeEnvironment });
-		task.presentationOptions = { panel: TaskPanelKind.Shared, reveal: /*TaskRevealKind.Silent*/ TaskRevealKind.Always };
 		
 		if (group != null) {
 			
 			task.group = group;
 			
 		}
 		
-		task.problemMatchers = [ "$haxe" ];
+		task.problemMatchers = vshaxe.problemMatchers.get ();
+
+		var presentation = vshaxe.taskPresentation;
+		task.presentationOptions = {
```

Comment:
We have to do it like this because the `vshaxe.taskPresentation` struct fields are read-only.